app/views/core/CreateAccountModal/CreateAccountModal.coffee

```diff
@@ -82,7 +91,9 @@ module.exports = class CreateAccountModal extends ModalView
     switch startOnPath
       when 'student' then @signupState.set({ path: 'student', screen: 'segment-check' })
       when 'individual' then @signupState.set({ path: 'individual', screen: 'segment-check' })
-      when 'teacher' then @signupState.set({ path: 'teacher', screen: if @euConfirmationRequiredInCountry() then 'eu-confirmation' else 'basic-info' })
+      when 'teacher'
+        startSignupTracking()
+        @signupState.set({ path: 'teacher', screen: if @euConfirmationRequiredInCountry() then 'eu-confirmation' else 'basic-info' }) # MAKING A NOTE HERE
```

Comment:
Do we need this comment here?

Reply:
Not as written.  This is either a typo, or needs to be updated to be meaningful.

Reply:
This is a typo.